ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/data.ts

```diff
@@ -0,0 +1,72 @@
+import { auth } from '@clerk/nextjs';
```

Comment:
This is technically unrelated, but I broke the shared vercel data fetch out into a separate file because the vercel build will complain that we're exporting non page related stuff from a vercel page.